ext/kv/remote.rs

```diff
@@ -126,7 +126,7 @@ impl<P: RemoteDbHandlerPermissions + 'static> DatabaseHandler
     };
 
     let options = &self.http_options;
-    let client = create_http_client(
+    let client = create_http_client_without_middleware(
```

Comment:
We're missing kv middleware. I keep `reqwest::Client` here since `denokv_remote` wants one.